workspaces/jupyterlab-python/install.sh

```diff
@@ -2,18 +2,42 @@
 
 set -ex
 
+# This script requires root.
+[ "$(id -u)" -eq 0 ] || exit 1
+
+# On PL, we want to standardize on using 1001:1001 for the user. We change
+# the "jovyan" account's default UID and GID here.
+USER_NAME=jovyan
+GROUP_NAME=$(id -gn $USER_NAME)
+OLD_UID=$(id -u $USER_NAME)
+OLD_GID=$(id -g $USER_NAME)
+NEW_UID=1001
+NEW_GID=1001
+groupmod -g $NEW_GID $GROUP_NAME
+usermod -u $NEW_UID -g $NEW_GID $USER_NAME
+for d in /home /tmp ; do
+    find $d -user $OLD_UID -execdir chown -h $NEW_UID {} +
+    find $d -group $OLD_GID -execdir chgrp -h $NEW_GID {} +
+done
+
 # Install dependencies and various libraries
 apt-get update && apt-get -y upgrade
-apt-get -y install gcc graphviz graphviz-dev
+apt-get -y install gosu gcc graphviz graphviz-dev
+
+# Test gosu and prepare the gosu helper.
+gosu jovyan true || exit 1
+mv /pl-gosu-helper.sh /usr/local/bin/
+chown root:root /usr/local/bin/pl-gosu-helper.sh
+chmod 0755 /usr/local/bin/pl-gosu-helper.sh
 
 # Install all Python dependencies.
-pip3 install -r /requirements.txt
+pip install -r /requirements.txt
```

Comment:
Nit: why the change from `pip3` to `pip`?

Reply:
With Python 2 in the past, "pip3" seems like an anachronism. It may not matter as long as the distro has both. On Ubuntu it looks like these are installed as two identical files (not necessarily hardlinked).

Reply:
In our other Dockerfiles and setup scripts, we either use `pip3` or `python3 -m pip ...`. I'd like to remain as consistent as possible (though I acknowledge that we're already inconsistent by having two ways of doing this).

Reply:
Went ahead and changed it back to pip3.